src/main/jbake/content/blog/2015/06/01/hawkular-metrics-0-3-4-released.adoc

```diff
@@ -0,0 +1,91 @@
+= Hawkular-Metrics 0.3.4 release and the way beyond
+Stefan Negrea
+2015-06-01
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, metrics, release
+
+
+I am happy to announce release 0.3.4 of Hawkular Metrics. The release is anchored by stability improvements and Docker + Kubernetes work.
+
+= Here are the release highlights:
+
+. InfluxDB Compatibility Layer
+* pretty printing of query results with the 'pretty' query parameter
+* absolute moment operands in queries (ex: select * from test where time > 1501560s and time < 4560546h)
+* various fixes which allow to integrate nicely with Grafana (declaring Hawkular Metrics as an InfluxDB 0.8 datasource)
+
+. RxJavaMigration
+
+* big refactoring to use RxJava instead of Guava's ListenableFuture
+* RxJava provides a much richer API for building asynchronous programs
+
+. Server Start-up
+
+* the server no longer requires Cassandra to be started first
+* any client requests will get back a 503 response code if the server has not fully initialized due to Cassandra
+being down at start up time
+
+. Docker and Kubernetes
+
+* all project docker images are now available in Docker Hub! Get them here:
+** https://registry.hub.docker.com/u/hawkular/hawkular-metrics/
+** https://registry.hub.docker.com/u/hawkular/hawkular-cassandra/
+* improved replication controller support now means that both the Hawkular-Metrics and Cassandra images can be scaled
+ up or down
+* various other smaller improvements, including better support for when containers startup
+
+. REST API (IMPORTANT!)
+
+* `tenantId` header was renamed to `Hawkular-Tenant`
+* `tenantId` header fallback was removed
+* `GET {id}/tags` only returns the tags, not the whole definition anymore
+* `GET {id}` returns the metric definition (see https://issues.jboss.org/browse/HWKMETRICS-110[HWKMETRICS-110] for
+known issue)
+
+. UI Explorer
+
+* was removed from the project
+** discussions are underway to find a new home for the functionality and code (probably hawkular-ui-components)
```

Comment:
@mtho11 Explained on irc: although we did migrate code from hawkular-ui-components to hawkular, I wouldn't call that repo totally abandoned:  ui -components that are not specific to the console should find their way back to that repo